api/metrics/rent_burdened_test.py

```diff
@@ -0,0 +1,77 @@
+import sys
+sys.path.append("../")
+
+from api.metrics.community import CommunityMetrics
+from api.utils.testing import create_test_db
+
+...
+
+def test_RBU():
+    rent_burdened_households_table = [
+        {
+            "area_number": 1,
+            "period_end_year": 2019,
+            "segment": "all",
+            "value": 43.31238192
+        },
+        {
+            "area_number": 1,
+            "period_end_year": 2018,
+            "segment": "all",
+            "value": 47.31238192
+        },
+        {
+            "area_number": 2,
+            "period_end_year": 2019,
+            "segment": "all",
+            "value": 57.77931
+        },
+        {
+            "area_number": 3,
+            "period_end_year": 2014,
+            "segment": "all",
+            "value": 32.31283212
+        },
+        {
+            "area_number": 3,
+            "period_end_year": 2018,
+            "segment": "all",
+            "value": 52.771
+        },
+        {
+            "area_number": 3,
+            "period_end_year": 2017,
+            "segment": "all",
+            "value": 53.64153641
+        }
+    ]
+    con, cur = create_test_db(
+        scripts=[
+            "./pipeline/load/rent_burdened_households.sql"
+        ],
+        tables={
+            "rent_burdened_households": rent_burdened_households_table
+        }
+    )
+
+    metric = RentBurdenedMetrics(con)
+    
+    assert metric.rent_burdened(year=2019, segment="all") == [
```

Comment:
Good unit tests! There are pros and cons to testing many functions in the same test case with the same input data:

- Pro: you only have to come up with one dataset
- Pro: if the dataset is simple, the reader can remember it when looking at each test
- Con: if one of the methods changes, you might have to rework the data while making sure you don't break the other tests
Con: if the dataset is complicated, the reader may have a harder time telling which inputs affect the output of each function

I think the two pros apply and the two cons are not as much of a worry in this case, so you can keep this test the way it is, just wanted to comment on this.